### ISSUE TITLE
Installing strongloop, can't install strong-mesh-client;;;; 

### ISSUE THREAD

**Issue by luncht1me** [strongloop/strong-mesh-client#47]
```
npm ERR! node v6.2.2
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE

npm ERR! strong-mesh-client@1.6.3 install: `node proxy/build-client.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the strong-mesh-client@1.6.3 install script 'node proxy/build-client.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the strong-mesh-client package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node proxy/build-client.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs strong-mesh-client
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls strong-mesh-client
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Guldan\Documents\GitHub\Select-Real-Estate\npm-debug.log
npm ERR! code 1
```

////

Windows 10 64bit, Python 2.7, MSVS 2015;

**Comment by ckrenn** [comment]
Any new info on this? I have the same issue on my Win10x64 machine during install.

**Comment by Abode113** [comment]
I think that (loopback) version is old version you need to install the newest version (loopback-cli)
so you need to type on terminal :
$ sudo npm install -g loopback-cli
and may be helpful to look at : https://loopback.io/doc/en/lb3/Command-line-tools.html

**Comment by rmg** [comment]
This should have been resolved with #58 and `strong-mesh-client@1.6.5`.

@luncht1me please re-open if you are still having this problem.
